stock_package_info/__manifest__.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# Copyright 2015-2018 Tecnativa - Pedro M. Baeza
+# Copyright 2015 AvanzOsc
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Package Info",
+    "version": "10.0.1.0.0",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "author": "AvanzOSC, "
+              "Tecnativa, "
+              "LasLabs, "
+              "Odoo Community Association (OCA)",
+    "website": "https://odoo-community.org/",
```

Comment:
@lasley URL should be `https://github.com/OCA/stock-logistics-workflow`

Reply:
@nikul-serpentcs : @lasley is no more working on OCA projects. I'm currently making household on logistics repositories and closing old PR's

Reply:
> @nikul-serpentcs : @lasley is no more working on OCA projects. I'm currently making household on logistics repositories and closing old PR's

@rousseldenis Ok It's Not blocking Issues, You can Continue your work, If Time then Improve it :smile: